thesheriff/application/raid/grade_raid.py

```diff
@@ -10,19 +13,23 @@ class GradeRaid:
     :type outlaw_repository: OutlawRepository
     """
 
-    def __init__(self, outlaw_repository: OutlawRepository):
+    @inject.autoparams()
+    def __init__(self, raid_repository: RaidRepository,
+                 outlaw_repository: OutlawRepository):
+        self.__raid_repository = raid_repository
         self.__outlaw_repository = outlaw_repository
 
-    def execute(self, raid: Raid) -> float:
+    def execute(self, raid_id: int) -> float:
```

Comment:
Not sure, I remember us deciding not to create request objects when it had only an id.
Otherwise, I like the idea because it is more extensible in case we need to add another parameter in the future.

Reply:
I thought in the same way, to gain extensibility

Reply:
+1 to GradeRaidRequest